doc/sphinx/language/core/basic.rst

```diff
@@ -499,6 +502,18 @@ The following attribute is supported by every command:
 
    Alias of :attr:`warnings`.
 
+Document-level attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. cmd:: Library Attributes {* #[ {*, @attribute } ] }
+   :name: Library Attributes
```

Comment:
The doc below should list the supported attributes (currently ``:attr:`deprecated` ``).

Reply:
OK, will do.